### PR TITLE
Change the y so that users can reach the bottom of the screen

### PR DESCRIPTION
**Pull request by CarolineLaw**

### CODE REVIEW

File: app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 import androidx.lifecycle.ViewModel
 import com.google.ar.core.AugmentedFace
 import com.google.ar.sceneform.math.Vector3
+import com.willowtree.vocable.R
 import com.willowtree.vocable.utils.VocableSharedPreferences
 import kotlinx.coroutines.*
 import org.koin.core.KoinComponent
+import org.koin.core.get
 import org.koin.core.inject
+import java.util.*
```
Comment (MichaelWTA) [inline app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt:16]: That's a bunch of imports. But that's pretty typical for android, right?

File: app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt [hunk 2]
```diff
@@ -78,6 +82,10 @@ class FaceTrackingViewModel : ViewModel(), KoinComponent {
                         liveAdjustedVector.postValue(oldVector)
                     }
                     else -> {
+                        val isTablet = get<Context>().resources.getBoolean(R.bool.is_tablet)
+                        if (!isTablet) {
+                            y *= 2f
```
Comment (MichaelWTA) [inline app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt:87]: will this keep multiplying y by 2 every time we hit this method? So will y keep growing and growing?

File: app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt (hunk 1 above)
Comment (CarolineLaw) [reply app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt:16]: ahh i forgot to push cmd+option+L, i'll do that

File: app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt (hunk 2 above)
Comment (CarolineLaw) [reply app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt:87]: I don't think so, also you should pull this down and test it on your own device if u can!

File: app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt [hunk 3]
```diff
@@ -78,6 +82,10 @@ class FaceTrackingViewModel : ViewModel(), KoinComponent {
                         liveAdjustedVector.postValue(oldVector)
                     }
                     else -> {
+                        val isTablet = get<Context>().resources.getBoolean(R.bool.is_tablet)
```
Comment (nicholas-cook) [inline app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt:85]: We should grab this value in the `init` block. Reading from the resources files can be expensive, and we don't want to do it every time we loop through here.

**Pull request merged**